syntax = "proto3";

package lorasensornetwork;

message Packet {

  bytes src_id =1 ;
  bytes dest_id = 2;
  MessageType msg_type = 3;
  bytes payload = 4;

  enum MessageType {
    JOIN_REQUEST = 0;
    JOIN_ACK = 1;
    SENSOR_RESPONSE = 2;
    SENSOR_REQUEST = 3; 
  }

}
